src/modules/navigator/rtl.h

```diff
@@ -49,6 +49,23 @@ class Navigator;
 class RTL final : public MissionBlock
 {
 public:
+	enum RTLState {
```

Comment:
Why is RTLState now public?

Reply:
Maybe I used it at some point somewhere else, but changed it again to private now.